CaloMC/src/CaloPhotonPropagation.cc

```diff
@@ -64,10 +66,11 @@ namespace mu2e {
    //----------------------------------------------------------------------------
    float CaloPhotonPropagation::propTimeSimu(float z)
    {       
-       int      iz   = int(z/dzTime_);       
+       unsigned iz   = z/dzTime_;
```

Comment:
This calculation may suffer from finite precision issues and can produce an out of bounds value.  is that the issue?